src/printer.js

```diff
@@ -1919,6 +1920,10 @@ function printArgumentsList(path, options, print) {
 
   if (groupLastArg) {
```

Comment:
Do you think that I should move it? If so, is the naming `groupLastArg` still relevant?